edge-router-avahi.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/python
+# -*- Mode:python; coding: utf-8; c-file-style:"gnu"; indent-tabs-mode:nil -*- */
+#
+# Copyright (C) 2016 Endless Computers INC.
+# Author: Niv Sardi <xaiki@endlessm.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+# A copy of the GNU Lesser General Public License is in the file COPYING.
+
+import logging
+
+import gi
+
+from gi.repository import GLib
+from gi.repository import GObject
+#from gi.repository import Gio
+
+from eos_data_distribution.MDNS import ServiceDiscovery
+
+logging.basicConfig(level=logging.INFO)
+
+SERVICES = [
+    # Disable TCP, we really only want UDP or ethernet
+    # "_nfd._tcp",
+    "_nfd._udp"]
```

Comment:
> not tested, but i'd say udp will work better for us... because, tcp is evil ?

What’s the reasoning here? It would be useful to have the reasoning in the comment in the code here.